Sources/Core/JavaModelRenderer.swift

```diff
@@ -35,7 +35,7 @@ public struct JavaModelRenderer: JavaFileRenderer {
             param.snakeCaseToPropertyName()
         }.joined(separator: ",\n")
 
-        return JavaIR.method(annotations: ["Override"], [.public], "int hashCode()") { [
+        return JavaIR.method(annotations: [JavaAnnotation.override], [.public], "int hashCode()") { [
```

Comment:
you should be able to remove `JavaAnnotation` and just put `.override` here and elsewhere . only reason to do this is if the compiler is unable to infer the value types

Reply:
This can be updated in a followup PR so we can merge this

Reply:
Ok will fix in a separate PR. I also realized I missed converting the string-based annotations in `JavaIR.Property` so I'll fix that too.